instrumentation.node.ts

```diff
@@ -1,27 +1,13 @@
-import { metrics } from "@opentelemetry/api";
-import { OTLPMetricExporter } from "@opentelemetry/exporter-metrics-otlp-http";
-import { Resource } from "@opentelemetry/resources";
-import { PeriodicExportingMetricReader } from "@opentelemetry/sdk-metrics";
-import { NodeSDK } from "@opentelemetry/sdk-node";
-import { SEMRESATTRS_SERVICE_NAME } from "@opentelemetry/semantic-conventions";
+import {
+	logRecordProcessor,
+	metricReader,
+	noopSpanProcessor,
+} from "@/lib/opentelemetry";
+import { registerOTel } from "@vercel/otel";
 
-// Configure OTLP metrics exporter
-const otlpMetricsExporter = new OTLPMetricExporter({
-	url: "https://ingest.us.signoz.cloud:443/v1/metrics",
-	headers: {
-		"signoz-access-token": process.env.SIGNOZ_INGESTION_TOKEN,
-	},
+registerOTel({
```

Comment:
use `registerOTel()` to simplify SDK configuration

without explicit configuration of OTEL Resource, we can gather useful info such as:
- Vercel build ID
- service version
- (and more)